code/datums/controllers/process/mobs.dm

```diff
@@ -53,8 +53,8 @@
 				M.Life(src)
 				if (!(c++ % 5))
 					scheck()
-			else if(istype(X, /mob/wraith))
-				var/mob/wraith/W = X
+			else if(istype(X, /mob/living/intangible/wraith))
```

Comment:
you can remove this `else if` now, since it'll get ticked in the `/mob/living/` check above